policy/SuprimecamMapper.paf

```diff
@@ -750,11 +701,9 @@ datasets: {
     # Measurements on coadds
 
     # Reference catalog for coadds
+    # dataset defined in obs_base modified in this mapper as follows:
```

Comment:
This `python` entry is definitely incorrect: it refers to the base class of the config class we want.  I suspect `obs_base` has the right one.